packages/search-ui-app-search-connector/src/requestAdapters.js

```diff
@@ -16,16 +16,21 @@ function rollup(f) {
   }));
 
   return {
-    [f.type || "any"]: values
+    [f.type || "all"]: values
```

Comment:
I couldn't reason about why this changed, but I think after you update the tests it might make more sense.